### DISCUSSION TITLE
2023-11-15

### DISCUSSION THREAD

**Discussion by miniposi** [10-rolling/rollingPolling#95] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] **다같이 merge~**
- [x] 충돌 오류 해결
- [x] reaction 누를 때 반응 안되는 문제 해결
- [x] 메세지 지우기 반응 안되는 문제 해결
- [x] 반응형 페이지 추가
- [x] 기타 수정사항
- [x] edit 페이지 무한 스크롤 적용
- [x] list 페이지 반응

### 📝 회의내용

* 오늘의 TMI ヾ(•ω•`)o
  수영님: 춥다..
  슬희님: 4시간 밖에 못 잠 -> _불면증_
  윤혁님: 피곤해서 아침 운동 못 감 !!
  수: 조금 피곤...
  
* 현재 진행 상황 공유
  거의 완성 !

* 할 일
- [x]   카카오톡 공유하기에 배포한 주소로 변경
- [x] 반응형 고치기 (윤혁님)
- [x] 로딩  (궁수)
- [x] 에러 처리 (수영님)

### 💪🏻 오늘 할일

* 알잘딱깔센
* **하늘 5번 보기**

### 🤔 특이사항

요새 오랜 시간 함께 하면서 부쩍 돈독해진 듯 합니다
마지막까지 파이팅 ! (ง •_•)ง

**Comment by hanseulhee** [comment]
수돈독 .... ❤️
